### PR TITLE
Added more details tot he spfct type and spfct_backslash method

### PR DESCRIPTION
**Pull request by abuttari**

### CODE REVIEW

**Comment by amontoison** [comment]
@abuttari give me the green light when this is ready to go.

**Review by amontoison — APPROVED** [review]

File: src/wrapper/qr_mumps_api.jl [hunk 1]
```diff
@@ -347,6 +347,45 @@ for (fname, lname, elty, subty) in (("sqrm_spbackslash_c", libsqrm, Float32   ,
     end
 end
 
+for (fname, lname, elty, subty) in (("sqrm_spfct_backslash_c", libsqrm, Float32   , Float32),
+                                    ("dqrm_spfct_backslash_c", libdqrm, Float64   , Float64),
+                                    ("cqrm_spfct_backslash_c", libcqrm, ComplexF32, Float32),
+                                    ("zqrm_spfct_backslash_c", libzqrm, ComplexF64, Float64))
+    @eval begin
+        function qrm_spbackslash!(spfct :: qrm_spfct{$elty}, b :: Vector{$elty}, x :: Vector{$elty})
```
Comment (amontoison) [inline src/wrapper/qr_mumps_api.jl:355]: `qrm_spfct_spbackslash!` ?

Comment (abuttari) [reply src/wrapper/qr_mumps_api.jl:355]: @amontoison you make me realize that all of these qrm functions modify the b right-hand side. should we remove all the methods without ! or for all of them, make a temporary copy of b?

Comment (amontoison) [reply src/wrapper/qr_mumps_api.jl:355]: For all the methods without ! we should make a temporary copy of b.
For all the methods with ! we just need to precise in the documentation that b is also updated.

**Comment by amontoison** [comment]
Thanks Alfredo !

**Pull request merged**